fs/accounting/accounting.go

```diff
@@ -61,7 +61,10 @@ func newAccountSizeName(stats *StatsInfo, in io.ReadCloser, size int64, name str
 		exit:   make(chan struct{}),
 		avg:    0,
 		lpTime: time.Now(),
-		max:    int64(fs.Config.MaxTransfer),
+		max:    -1,
+	}
+	if fs.Config.CutoffMode != fs.CutoffModeSoft {
```

Comment:
Should this be `CutoffMode == CutoffModeHard`? Cautious should stop before it gets to the limit shouldn't it?